src/main/java/com/kakao/stringadder/domain/splitter/ExpressionParser.java

```diff
@@ -0,0 +1,41 @@
+package com.kakao.stringadder.domain.splitter;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ExpressionParser {
+    private static final String CUSTOM_PARSE_PATTERN = "//(.)\n(.*)";
+    private static final Pattern regexPattern = Pattern.compile(CUSTOM_PARSE_PATTERN);
+    private static final DefaultSplitter defaultSplitter = new DefaultSplitter();
+
+    private ExpressionParser() {
+    }
+
+    public static List<String> parse(String calculationInput) {
+        Matcher matcher = regexPattern.matcher(calculationInput);
+        ParserResult result = findPattern(matcher);
+        if (result != null)
+            return new CustomSplitter(result.delimiter).split(result.input);
```

Comment:
```suggestion
        if (result != null) {
            return new CustomSplitter(result.delimiter).split(result.input);
        }
```
한 줄이라도 중괄호는 생략하지 않는 것이 좋습니다 :)